index.js

```diff
@@ -0,0 +1,3 @@
+'use strict'
+
+module.exports = {}
```

Comment:
What is the purpose of this file?

Reply:
I was originally having difficulties using this project with Typescript *without* a `main` file. Also, the`index.js` was already referenced in the original `package.json`. I've resolved the issues on my end and removed this file and its reference.